src/player.cpp

```diff
@@ -9190,31 +9190,6 @@ bool player::is_waterproof( const std::bitset<num_bp> &parts ) const
     return covered_with_flag("WATERPROOF", parts);
 }
 
-bool player::has_amount(const itype_id &it, int quantity) const
-{
-    if (it == "toolset")
-    {
-        return has_active_bionic("bio_tools");
-    }
-    return (amount_of(it) >= quantity);
-}
-
-int player::amount_of(const itype_id &it) const
-{
-    if (it == "toolset" && has_active_bionic("bio_tools")) {
-        return 1;
-    }
-    if (it == "apparatus") {
-        return ( has_items_with_quality("SMOKE_PIPE", 1, 1) ? 1 : 0 );
```

Comment:
@mugling removing this broke the smoking pipes. Could you give me a clue how to fix this simply? (or feel free to fix it, if you have time and desire to do so).